metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -432,8 +432,8 @@ private Throwable handleEventException(String name,
                                            OptionalLong startProcessingTimeNs,
                                            Throwable exception) {
         if (!startProcessingTimeNs.isPresent()) {
-            log.error("{}: unable to start processing because of {}.", name,
-                exception.getClass().getSimpleName());
+            log.error("{}: unable to start processing because of {}. Reason: {}", name,
```

Comment:
Yes, but unsure if that would be too many logs because it'll also log the stack trace. WDYT?

Reply:
From a debugging point of view, I think that the stack trace may be useful if it is not an instance of a known exception, otherwise the error message is sufficient.

Reply:
That's reasonable.